crates/holochain/src/conductor/handle.rs

```diff
@@ -894,31 +902,75 @@ impl ConductorHandleT for ConductorHandleImpl {
     async fn create_clone_cell(
         self: Arc<Self>,
         payload: CreateCloneCellPayload,
-    ) -> ConductorResult<CellId> {
+    ) -> ConductorResult<InstalledCell> {
         let CreateCloneCellPayload {
-            properties,
-            dna_hash,
-            installed_app_id,
-            agent_key,
+            app_id,
             role_id,
+            properties,
+            network_seed,
             membrane_proof,
+            origin_time,
+            name,
         } = payload;
-        let cell_id = CellId::new(dna_hash, agent_key);
-        let cells = vec![(cell_id.clone(), membrane_proof)];
-
-        // Run genesis on cells.
-        crate::conductor::conductor::genesis_cells(&self.conductor, cells, self.clone()).await?;
+        if network_seed == None && properties == None {
+            return Err(ConductorError::CloneCellError(
+                "neither network_seed nor properties provided for clone cell".to_string(),
+            ));
+        }
```

Comment:
Interestingly, you would also get a different hash if `origin_time` were modified too. So maybe that should be included in the check. If there were a `DnaPhenotypeOption` struct, there could be a method on the struct that just checks if any of the fields are `Some`.